### ISSUE TITLE
SimpleReport home page needs a `<main>` landmark

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4034]
## Area or Component
Static site homepage - simplereport.gov

## Quick Summary
The home page doesn't have a `<main>` section in the DOM.

## Accessibility Violation
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

There are a host of related issues, as each individual element not contained by a landmark generates its own issue that `All page content should be contained by landmarks`

## Action Requested
Add a `<main>` landmark to the home page

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=22:22
